README.md

```diff
@@ -10,13 +10,21 @@ non-public information about using this repository.
 
 ### Terraform
 
-The main terraform directories are:
+Our Terraform code is organized by two concepts, with two corresponding directories.
 
-* `modules`: where we decompose our configuration into [Terraform
-  modules](https://www.terraform.io/docs/configuration-0-11/modules.html)
-* `modules/stack/base` & `modules/stack/spoke`: the main modules that define
-  the bulk of each environment
-* `stacks`: the various "environments"
+* **Modules** are reusable units of terraform code. Each module describes a single useful concept in our infrastructure. Modules are parameterized by terraform variables for reuse, and are located in `terraform/modules`.
+  * Two important modules are `modules/stack/base` and `modules/stack/spoke`.
+  * Read more about Terraform modules: https://developer.hashicorp.com/terraform/language/modules
+* **Stacks** combine and configure modules for use in an environment. They are also parameterized by variables, such as the name of the environment. Stacks are located in `terraform/stacks`.
```

Comment:
Wording change:

They are also parameterized by variables, with values such as the environment name.